lib/reversetunnel/transport.go

```diff
@@ -314,6 +357,18 @@ func (p *transport) start() {
 
 	errorCh := make(chan error, 2)
 
+	if signedHeader != nil {
+		p.log.Debugf("Transport writing signed PROXY header to the remote connection %s -> %s", conn.LocalAddr(), conn.RemoteAddr())
+		_, err = conn.Write(signedHeader)
+		if err != nil {
+			p.log.Errorf("could not write PROXY header to the connection: %v", err)
```

Comment:
Nit: log capitalisation:
```suggestion
			p.log.Errorf("Could not write PROXY header to the connection: %v", err)
```